src/core/main.js

```diff
@@ -580,8 +580,18 @@ p5.prototype._initializeInstanceVariables = function() {
 // global mode.
 p5.instance = null;
 
-// Allows for the friendly error system to be turned off when creating a sketch,
-// which can give a significant boost to performance when needed.
+/**
+ * Allows for the friendly error system (FES) to be turned off when creating a sketch,
+ * which can give a significant boost to performance when needed.
+ * See <a href='https://github.com/processing/p5.js/wiki/Optimizing-p5.js-Code-for-Performance#disable-the-friendly-error-system-fes'>
+ * disabling the friendly error system</a>.
+ *
+ * @property {Boolean} disableFriendlyErrors
+ * @example
+ * <div class="norender notest"><code>
+ * p5.disableFriendlyErrors = true; // Disables FES
+ * </code></div>
```

Comment:
can you add a setup() function here with just a createcanvas() line, so the user can see where this line goes in relation to setup?

Reply:
Updated, is this what you were looking for?

Reply:
great thanks